src/debug/createdump/datatarget.cpp

```diff
@@ -151,7 +151,7 @@ DumpDataTarget::ReadVirtual(
 {
     assert(m_fd != -1);
     size_t read = pread64(m_fd, buffer, size, (off64_t)(ULONG_PTR)address);
-    if (read == -1)
+    if (read == (size_t)-1)
```

Comment:
A nit - make read ssize_t instead (it is actually the real return type of pread64), you won't need the cast then.